operate/client/src/App/Processes/MigrationView/BottomPanel/index.tsx

```diff
@@ -125,6 +127,23 @@ const BottomPanel: React.FC = observer(() => {
               const selectableTargetFlowNodes =
                 processXmlMigrationTargetStore.selectableFlowNodes.filter(
                   (flowNode) => {
+                    /**
+                     * For boundary events allow only target flow nodes with the same event type
+                     */
+                    if (
+                      hasType({
+                        businessObject: sourceFlowNode,
+                        types: ['bpmn:BoundaryEvent'],
+                      })
+                    ) {
+                      return (
+                        getEventType(sourceFlowNode) === getEventType(flowNode)
+                      );
+                    }
```

Comment:
This ensures that I can only map message events to message events and timer events to timer events in the instance migration view:

![image](https://github.com/user-attachments/assets/61dea89f-9301-45bb-ae50-668f5e4d26b6)

![image](https://github.com/user-attachments/assets/c34d6ccb-68e4-4c86-8c72-8a4b2599cd9c)